### DISCUSSION TITLE
Removing the dependency on unpkg.com

### DISCUSSION THREAD

**Discussion by hutbert** [Laravel-Backpack/community-forum#1203] [General] [answered]
We have a few sites that are unusable today because unpkg.com is down. Is there a way to rebuild the javascript assets to remove the dependency on unpkg.com? This caught us off guard and clients are unhappy.

**Comment by pxpm** [comment, marked as answer]
Hey @hutbert 
At the moment the only way would be to overwrite all the views where unkpkg.com is used, and change it to ther cdn, or your own downloaded files. 

Unfortunately it's not the first time that bites us, so we are already working on Basset v2, that will have a "easy" way to change the assets without overwriting any views, plus you can cache locally and push to prod (that is already possible, but probably not documented), and there are some use-cases that are not fully covered, like assets that depend on some runtime value. 

All this has been taken care for the next version. I see the cdn is already up, it could be some momentary hiccup, but nonetheless it's something we are working to avoid in the next version, due in a few months. (hopefully before new year). 

Cheers

**Reply by hutbert** [reply]
@pxpm a local cache would be great - glad that is in the work. Thanks for the quick response!

**Comment by yurik94** [comment]
unpkg.com are again slow/down, we have also paid for a pro license, which is useless at the moment.
Customer of course is complaining, we would like to have an easy way to replace it, even by serving the files by ourself as we are using S3 + CloudFront.

**Reply by nekhbet** [reply]
same here, I don't understand why for the sake of "simplicity" people use CDNs hosted in garages that are more down than up
